src/utils/logger.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var PREFIX = '[algoliasearch-helper]';
+
+function warn(message) {
+  console.warn(PREFIX + ' ' + message);
+}
+
+module.exports = {
+  warn
```

Comment:
```suggestion
  warn: warn
```

this is causing the build error

Reply:
oh.. I've forgotten about this for a long time.